### PR TITLE
Feature/apl 1691 1692

### PR DESCRIPTION
**Pull request by alzinchenko**
TPS optimization

### CODE REVIEW

**Review by oleksiylukin — APPROVED** [review]

File: apl-conf/src/main/resources/conf-tn3/apl-blockchain.properties [hunk 1]
```diff
@@ -118,7 +118,7 @@ apl.pullThreshold=0
 apl.blacklistingPeriod=600000
 
 # Consider a new transaction or block sent after 10 peers have received it.
-apl.sendToPeersLimit=10
+apl.sendToPeersLimit=3
```
Comment (andrii-firstbridge) [inline apl-conf/src/main/resources/conf-tn3/apl-blockchain.properties:121]: As I see property has been changed only for tn-3. Are you going to change it for the tn-1/2 and mainnet?

File: apl-conf/src/main/resources/conf-tn3/apl-blockchain.properties [hunk 2]
```diff
@@ -471,7 +471,7 @@ apl.enableTransactionRebroadcasting=true
 # transactions to include in a forged block is used to decide which to keep in
 # the unconfirmed pool.
 # Set to 0 or negative for no limit.
-apl.maxUnconfirmedTransactions=2000
+apl.maxUnconfirmedTransactions=10000
```
Comment (andrii-firstbridge) [inline apl-conf/src/main/resources/conf-tn3/apl-blockchain.properties:474]: same as above.
\

**Review by yuriy-larin-firstbridge — APPROVED** [review]

**Review by andrii-firstbridge — APPROVED** [review]

File: apl-conf/src/main/resources/conf-tn3/apl-blockchain.properties (hunk 2 above)
Comment (AndrewBoyarsky) [reply apl-conf/src/main/resources/conf-tn3/apl-blockchain.properties:474]: Fixed

File: apl-conf/src/main/resources/conf-tn3/apl-blockchain.properties (hunk 1 above)
Comment (AndrewBoyarsky) [reply apl-conf/src/main/resources/conf-tn3/apl-blockchain.properties]: Fixed

**Pull request merged**